lnwallet/btcwallet/psbt.go

```diff
@@ -211,11 +215,11 @@ func (b *BtcWallet) SignPsbt(packet *psbt.Packet) error {
 				"PSBT signing: %v", signMethod)
 		}
 		if err != nil {
-			return err
+			return signedInputs, err
```

Comment:
I think it's best practice to return `nil` (if it's a nillable type) in the error case.